docs/content/blog/v0.10.fa.md

```
@@ -0,0 +1,247 @@
+---
+
+Hextra v0.10.0 یک انتشار بزرگ است که مملو از قابلیت‌های جدید، ارتقاهای معماری و بهبودهای کیفیت زندگی می‌باشد.
+
+<!--more-->
+
+همچنین شامل مشارکت‌های 10 [مشارکت‌کننده جدید](#contributors) بوده و درخواست‌های دیرینه جامعه را برطرف می‌کند.
+
+## راهنمای ارتقا
+
+> [!IMPORTANT]
+> **تغییرات شکست‌آمیز**: این انتشار شامل چندین تغییر شکست‌آمیز است. لطفاً قبل از ارتقا، چک‌لیست و [راهنمای مهاجرت](#migration-guide) را بررسی کنید.
```

markdownlint-cli2 (0.17.2)

Link fragments should be valid

(MD051, link-fragments)